### PR TITLE
Tornike/legals update

### PR DESCRIPTION
**Pull request by tkeburia**
- [x] Create content for privacy policy page
- [x] Create content for terms of use page
- [x] update links to new pages

### CODE REVIEW

File: src/pages/privacy-policy.tsx [hunk 1]
```diff
@@ -1,32 +1,516 @@
...
+          </li>
+          <li>
+            information about your use of or delivery of Services including
+            details of the content you viewed on our Platform, how long you
+            stayed and on which pages, and your navigation around the Platform;
+            and
+          </li>
+          <li>
+            other data to ensure and check compliance with the Platform terms of
+            use [insert link] (<strong>“Terms of Use”</strong>) and/or otherwise
```
Comment (s6577t) [inline src/pages/privacy-policy.tsx]: need an actual link here

File: src/pages/privacy-policy.tsx [hunk 2]
```diff
@@ -1,32 +1,516 @@
...
+          Compliance Manager in writing using the following contact details:{" "}
+          <a href="mailto:barrymurphy@esma.community">
+            barrymurphy@esma.community
+          </a>
+          . The tasks of our Data Compliance Manager include (for example)
+          monitoring our compliance with applicable data protection laws and
+          acting as contact for individuals whose data is processed by us.
+        </p>
+
+        <ol start={21}>
```
Comment (s6577t) [inline src/pages/privacy-policy.tsx]: 22?

**Comment by s6577t** [comment]
We can delete the PDFs from `/static` now too

**Review by s6577t — CHANGES_REQUESTED** [review]

File: src/pages/privacy-policy.tsx (hunk 1 above)
Comment (tkeburia) [reply src/pages/privacy-policy.tsx]: oops

**Review by s6577t — APPROVED** [review]

**Pull request merged**